tabpy-server/tabpy_server/common/default.conf

```diff
@@ -1,11 +1,42 @@
+[TabPy]
 TABPY_QUERY_OBJECT_PATH = /tmp/query_objects
 TABPY_PORT = 9004
 TABPY_STATE_PATH = ./
 TABPY_BIND_IP = 127.0.0.1
-TABPY_LOG_LEVEL = INFO
 
 # To set up secure TabPy uncomment and modify the following lines.
 # Note only PEM-encoded x509 certificates are supported
 # TABPY_TRANSFER_PROTOCOL = https
 # TABPY_CERTIFICATE_FILE = path/to/certificate/file.crt
 # TABPY_KEY_FILE = path/to/key/file.key
+
+[loggers]
```

Comment:
This is good but probably deserves a separate file (?) I don't think the startup script knows to read this

Reply:
We actually want to move all the configurations (state.ini as well) to one config file for simplifying maintenance